src/recursion.js

```diff
@@ -6,174 +6,174 @@
 // denoted by n!, is the product of all positive integers less than or equal to n.
 // Example: 5! = 5 x 4 x 3 x 2 x 1 = 120
 // factorial(5); // 120
-let factorial = function(n) {
+function factorial(n) {
```

Comment:
I wouldn't say this syntax is more standard, but simply different syntax for a different purpose. The existing syntax is known as [function expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function) and your changes are [function declarations](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function). The primary reason to change from expressions to declarations is [hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) which doesn't really matter here due to the scope.

The reason I lean toward leaving these as expressions is because the variables are [temporarily reassigned](https://github.com/JS-Challenges/recursion-prompts/blob/27af23edf0a673da4765e7d47ac7c9571e04e3c0/spec/part1.js#L13) in the specs. Since we're reassigning the variables, it feels more consistent to instantiate them as variables.

If your intent is to modernize this code, you might consider refactoring these boilerplate function signatures to [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

Reply:
Thanks for the links and review!

> If your intent is to modernize this code, you might consider refactoring these boilerplate function signatures to [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

I don't see arrow functions as the modern way to define functions, I feel as though their purpose are  anonymous functions, and not to replace the `function` keyword entirely.

Reply:
Seems like we just have a different way to see this, even though I definitely see your point of view!

Reply:
I wasn't suggesting that arrow functions are "more standard" or better, but simply more _modern_ since the `function` keyword predates arrow functions. Modern doesn't inherently mean better. For the context of this repo, the only reason to choose one of these function syntaxes over the other is personal preference. You seemed to be motivated to contribute, so I was simply offering a suggestion for a way you could contribute that aligns with existing patterns in this repo.

If you think of other ways you'd like to contribute, I welcome your input. I applaud you for being proactive.